packages/lit-dev-content/site/docs/v3/tools/requirements.md

```diff
@@ -43,153 +43,8 @@ Webpack automatically handles bare module specifiers; for Rollup, you'll need a
 
 ### Modern browser breakdown
 
-All modern browsers update automatically and users are highly likely to have a recent version. The following table lists the minimum version of each major browser that natively supports ES2019 and web components, the key features on which Lit relies.
+All modern browsers update automatically and users are highly likely to have a recent version. Lit and related libraries are tested on the latest minus 2 major versions of Chromium and Safari, and Firefox Extended Support Release. Older versions may still work but will be at best efforts without guarantee.
```

Comment:
instead of "are tested on the latest minus 2 major versions of Chrome and Safari, suggest:

"are tested on the latest two major versions of..."